### PR TITLE
Und 633

### PR DESCRIPTION
**Pull request by Ashish-srij**
Header on all templates

### CODE REVIEW

File: stories/Templates/Articlepage/ArticlePage.jsx [hunk 1]
```diff
@@ -33,24 +33,34 @@ const ArticlePage = ({
   footerData,
   headingLabel,
   featuredCardContent,
+
```
Comment (shishirphadke) [inline stories/Templates/Articlepage/ArticlePage.jsx:36]: @Ashish-srij Remove empty line.

File: stories/Templates/Articlepage/ArticlePage.jsx [hunk 2]
```diff
@@ -33,24 +33,34 @@ const ArticlePage = ({
...
   languageswitcherData,
+  navigationData,
   rightNavigationData,
   leftNavigationData,
-  navigationData,
   locale,
+  getMegaMenuData,
```
Comment (shishirphadke) [inline stories/Templates/Articlepage/ArticlePage.jsx:42]: @Ashish-srij I do not see `getMegaMenuData` being used anywhere. Can you remove it if it is not required.

File: stories/Templates/AuthorPage/AuthorPage.jsx [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
+import { Footer } from "../../Organism/Footer/Footer";
+import "./AuthorPage.scss";
 
 const AuthorPage = ({
-  breadcrumbData, authorSummaryData, contentCardData, footerData,  languageswitcherData,
+  breadcrumbData,
+  authorSummaryData,
+  contentCardData,
+  footerData,
+
```
Comment (shishirphadke) [inline stories/Templates/AuthorPage/AuthorPage.jsx:18]: @Ashish-srij Remove empty line.

File: stories/Templates/AuthorPage/AuthorPage.jsx [hunk 4]
```diff
@@ -1,35 +1,47 @@
...
+  languageswitcherData,
+  navigationData,
   rightNavigationData,
   leftNavigationData,
-  navigationData,
   locale,
+  getMegaMenuData,
```
Comment (shishirphadke) [inline stories/Templates/AuthorPage/AuthorPage.jsx:24]: @Ashish-srij I do not see `getMegaMenuData` being used anywhere. Can you remove it if it is not required.

File: stories/Templates/Authors/Authors.jsx [hunk 5]
```diff
@@ -1,56 +1,70 @@
...
+  text,
+  authorData,
+  headingData,
+  languageswitcherData,
+  navigationData,
   rightNavigationData,
   leftNavigationData,
-  navigationData,
   locale,
+  getMegaMenuData,
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:34]: @Ashish-srij I do not see `getMegaMenuData` being used anywhere. Can you remove it if it is not required.

File: stories/Templates/Authors/Authors.jsx [hunk 6]
```diff
@@ -1,56 +1,70 @@
...
+  locationData,
+  langSelect,
+  backcaption,
 }) => {
   useEffect(() => {
-  console.log(text);
-    viewport('.feature__card--headertext');
+    console.log(text);
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:40]: @Ashish-srij Remove `console.log(text);`.

File: stories/Templates/Authors/Authors.jsx [hunk 7]
```diff
@@ -1,56 +1,70 @@
...
-import { Authorcard } from '../../Components/UIcomponents/Author/Authorcard/Authorcard';
-import { CtaButton } from '../../Components/UIcomponents/Buttons/Cta_button/CtaButton';
-import { SearchExpand } from '../../Components/Forms/expandable search/SearchExpand';
+import React, { useEffect } from "react";
+import logowhite from "../../assets/images/undp-logo-white.svg";
+import logo from "../../assets/images/undp-logo-blue.svg";
+import user from "../../assets/images/user.svg";
+import viewport from "../../assets/js/viewport";
+import { Heading } from "../../Atom/Typography/Heading/Heading";
+import CustomSelect from "../../Components/Forms/Dropdowns/CustomSelect/CustomSelect";
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:7]: @Ashish-srij I do not see `CustomSelect` being used anywhere. Can you remove the entire import statement if it is not required.

File: stories/Templates/Authors/Authors.jsx [hunk 8]
```diff
@@ -1,56 +1,70 @@
...
+import { Breadcrumbcomponent } from "../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import GlobalHeader from "../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader";
+import { AuthorSummary } from "../../Organism/Blocks/Authorcard/AuthorSummary";
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:10]: @Ashish-srij I do not see `AuthorSummary` being used anywhere. Can you remove the entire import statement if it is not required.

File: stories/Templates/Authors/Authors.jsx [hunk 9]
```diff
@@ -1,56 +1,70 @@
...
+import { ContentCard } from "../../Organism/Blocks/ContentCard/ContentCard";
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:11]: @Ashish-srij I do not see `ContentCard` being used anywhere. Can you remove the entire import statement if it is not required.

File: stories/Templates/Authors/Authors.jsx [hunk 10]
```diff
@@ -1,56 +1,70 @@
...
 const Authors = ({
-  breadcrumbData, authorSummeryData, contentCardData, footerData,
-  sortText, viewText, closeText,
-  undpheading, text, authorData, headingData, buttontype, buttonData,  languageswitcherData,
+  breadcrumbData,
+  footerData,
+  sortText,
+  viewText,
+  closeText,
+  undpheading,
```
Comment (shishirphadke) [inline stories/Templates/Authors/Authors.jsx:25]: @Ashish-srij I do not see `undpheading` being used anywhere. Can you remove it if it is not required.

File: stories/Templates/Articlepage/ArticlePage.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Templates/Articlepage/ArticlePage.jsx]: done

File: stories/Templates/Articlepage/ArticlePage.jsx (hunk 2 above)
Comment (Ashish-srij) [reply stories/Templates/Articlepage/ArticlePage.jsx]: done

File: stories/Templates/AuthorPage/AuthorPage.jsx (hunk 3 above)
Comment (Ashish-srij) [reply stories/Templates/AuthorPage/AuthorPage.jsx]: done

File: stories/Templates/AuthorPage/AuthorPage.jsx (hunk 4 above)
Comment (Ashish-srij) [reply stories/Templates/AuthorPage/AuthorPage.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 5 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 6 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 7 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 8 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 9 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: done

File: stories/Templates/Authors/Authors.jsx (hunk 10 above)
Comment (Ashish-srij) [reply stories/Templates/Authors/Authors.jsx]: @shishirphadke I have made all the changes.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss [hunk 11]
```diff
@@ -6,6 +6,11 @@
   background-color: $color-gray-100;
   box-shadow: 0 3px 6px rgba(0, 0, 0, 0.1);
   height: 7.1875rem;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  z-index: 9;
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss]: Done

**Pull request merged**